base/src/macros.rs

```diff
@@ -0,0 +1,13 @@
+
+#[macro_export]
+macro_rules! ice {
+    () => ({
+        ice!("ICE: Please report an issue at https://github.com/gluon-lang/gluon/issues")
```

Comment:
@bostelk @Marwes This should probably be "panic!", as otherwise the "Please report ..." will be printed twice.

Reply:
Oops, you're right. Fixed in #386.